specs/commandline/verify.md

```diff
@@ -0,0 +1,99 @@
+      --plugin-config strings   {key}={value} pairs that are passed as is to a plugin, if the verification is associated with a verification plugin, refer plugin documentation to set appropriate values
+  -u, --username string         Username for registry operations (default to $NOTATION_USERNAME if not specified)
+```
+
+## Usage
+
+Pre-requisite: User needs to configure trust store and trust policy properly before using `notation verify` command.
+
+### Configure Trust Store
+
+Use `notation certificate` command to configure trust stores.
+
+### Configure Trust Policy
+
+Users who consume signed artifact from a registry use the trust policy to specify trusted identities which sign the artifacts, and level of signature verification to use. The trust policy is a JSON document. User needs to create a file named `trustpolicy.json` under `{NOTATION_CONFIG}`. See [Notation Directory Structure](https://github.com/notaryproject/notation/blob/main/specs/directory.md) for `{NOTATION_CONFIG}`.
+
+An example of `trustpolicy.json`:
+
+```jsonc
+{
+    "version": "1.0",
+    "trustPolicies": [
+        {
+            // Policy for all artifacts, from any registry location.
+            "name": "wabbit-networks-images",  // Name of the policy.
+            "registryScopes": [ "*" ],         // The registry artifacts to which the policy applies.
+            "signatureVerification": {         // The level of verification - strict, permissive, audit, skip.
+                "level": "strict"
+            },
+            "trustStores": [ "ca:wabbit-networks" ], // The trust stores that contains the X.509 trusted roots.
+            "trustedIdentities": [                   // Identities that are trusted to sign the artifact.
+                "x509.subject: C=US, ST=WA, L=Seattle, O=wabbit-networks.io, OU=Finance, CN=SecureBuilder"
+            ]
+        }
+    ]
+}
+```
+
+In this example, only one policy is configured with the name `wabbit-networks-images`. With the value of property `registryScopes` set to `*`, this policy applies to all artifacts from any registry location. User can configure multiple trust policies for different scenarios. See [Trust Policy Schema and properties](https://github.com/notaryproject/notaryproject/blob/main/trust-store-trust-policy-specification.md#trust-policy) for details.
+
+### Verify signatures on a container image stored in a registry (Neither trust store nor trust policy is configured yet)
+
+```shell
+
+# Prerequisites: Signatures are stored in a registry referencing the signed container image
+
+# Configure trust store by adding a certificate file into trust store named "wabbit-network" of type "ca"
+notation certificate add --type ca --store wabbit-networks wabbit-networks.crt
+
+# Configure trust policy by creating a JSON document named "trustpolicy.json" under directory "{NOTATION_CONFIG}"
+# Example on Linux
+cat <<EOF > $HOME/.config/notation/trustpolicy.json
```

Comment:
This is going to vary based on the user's operating system.  IMO we should consider if we can get a "bare bones" cli implementation which simply writes a template similar to what's there to the user's proper directory (i.e. **notation policy create -n default**) and possibly opens it up or lists where it is at.  (i.e. **notation policy list** or **notation policy open** (optional **--name default**)

Reply:
I will create a new issue for a discussion regarding this request.

Reply:
https://github.com/notaryproject/notation/issues/398